slack/web/__init__.py

```diff
@@ -0,0 +1,20 @@
+import platform
+import sys
+
+import slack.version as ver
+
+
+def get_user_agent():
```

Comment:
Extracted to reuse in `UrllibWebClient` - the method needs to be outside the `BaseClient` to avoid circular import issues.